internal/static/internal/js/secrets.js

```diff
@@ -0,0 +1,13 @@
+const SECRET_SHOW_DURATION_SECONDS = 10;
+
+$(".secret-button").click(function () {
+    const secretButton = $(this);
+    const secretId = secretButton.data("secret-id");
+    const secret = $(`#${secretId}`);
+    secret.removeClass("hidden");
+    secretButton.addClass("hidden");
+    setTimeout(() => {æ
```

Comment:
I don't think the"æ" is supposed to be there, and it's stopping the timeout from working